test_plans/sub_group_mask.asciidoc

```diff
@@ -36,13 +48,73 @@ All of the following tests have these initial steps performed in defferent .cpp
 
 Check that member `size_t max_bits` exists.
 
-=== Tests for Member Functions
+=== Tests for constructors and operator= [[ctors_rev_2]]
+
+Features described in this paragraph are available from revision 2 of the
+extension.
+
+==== sub_group_mask() [[default_ctor]]
+
+* Construct `sub_group_mask` instance `s` `with sub_group_mask()` ctor.
+* Check that `s.none() == true`.
+
+==== sub_group_mask(unsigned long long val) [[ull_ctor]]
+
+* Construct `sub_group_mask` instance `s` with `sub_group_mask(unsigned long long val)` ctor
+  and `size_t val` == `[item.get_local_linear_id(), 0b1010...]`.
+* Check that for `N = 0...sub_group_mask.size() - 1` `s[id(N)]` == `((val >> N) & 1)`
```

Comment:
I'm sorry, but I don't understand this at all.  What does the notation with the square brackets mean:

> `size_t val` == `[item.get_local_linear_id(), 0b1010...]`

`val` is a scalar value, so I don't understand what the square brackets are supposed to signify.

Reply:
Corrected algorithm description.